app/src/main/java/com/shagalalab/constitution/article/ArticleViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package com.shagalalab.constitution.article
+
+import android.graphics.Typeface
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.text.style.StyleSpan
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.shagalalab.constitution.data.dao.ArticleDao
+import com.shagalalab.constitution.data.models.ArticleModel
+import java.util.concurrent.Executors
+
+class ArticleViewModel(private val articleDao: ArticleDao) : ViewModel() {
+
+    private val articleListLiveData: MutableLiveData<List<ArticleModel>> = MutableLiveData()
+    val articleList: LiveData<List<ArticleModel>> = articleListLiveData
+
+    fun getArticles(chapterId: Int) {
+        Executors.newSingleThreadExecutor().execute {
+            articleListLiveData.postValue(articleDao.getArticlesByChapterId(chapterId))
+        }
+    }
+
+    fun normalizedDescription(it: List<ArticleModel>): SpannableStringBuilder {
```

Comment:
Nah, this is not correct. It is not ViewModel's job to transform data. When I asked to move it to model, I meant `ArticleModel` class. Move this method to `ArticleModel`, it can transform only one article, you can combine them in Fragment itself.